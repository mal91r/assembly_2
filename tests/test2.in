isdfasdf9234928342.,/,?#$!(#$&*!(*%h
