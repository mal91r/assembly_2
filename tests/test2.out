ASCII:33 CHAR:! COUNT:2
ASCII:35 CHAR:# COUNT:2
ASCII:36 CHAR:$ COUNT:2
ASCII:37 CHAR:% COUNT:1
ASCII:38 CHAR:& COUNT:1
ASCII:40 CHAR:( COUNT:2
ASCII:42 CHAR:* COUNT:2
ASCII:44 CHAR:, COUNT:2
ASCII:46 CHAR:. COUNT:1
ASCII:47 CHAR:/ COUNT:1
ASCII:50 CHAR:2 COUNT:3
ASCII:51 CHAR:3 COUNT:2
ASCII:52 CHAR:4 COUNT:2
ASCII:56 CHAR:8 COUNT:1
ASCII:57 CHAR:9 COUNT:2
ASCII:63 CHAR:? COUNT:1
ASCII:97 CHAR:a COUNT:1
ASCII:100 CHAR:d COUNT:2
ASCII:102 CHAR:f COUNT:2
ASCII:104 CHAR:h COUNT:1
ASCII:105 CHAR:i COUNT:1
ASCII:115 CHAR:s COUNT:2
